AspNetCoreApplication/Repositories/IBookCategoryRepository.cs

```diff
@@ -0,0 +1,17 @@
+﻿using AspNetCoreApplication.Models;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace AspNetCoreApplication.Repositories
+{
+    public interface IBookCategoryRepository
+    {
+        Task Add(int bookId, int categoryId);
+
+        Task Delete(int bookId, int categoryId);
+
+        Task<List<Category>> GetCategories(int bookId);
```

Comment:
Move hàm này sang BookRepository
```suggestion
        Task<List<Category>> GetCategories(int bookId);
```

Reply:
done